docs/uploader/aws-s3-multipart.mdx

```diff
@@ -36,8 +36,8 @@ completion requests besides the upload requests. For example, if you are
 uploading files that are only a couple kilobytes with a 100ms roundtrip latency,
 you are spending 400ms on overhead and only a few milliseconds on uploading.
 
-If you are uploading large files (100&nbsp;MB+), we recommend
-`@uppy/aws-s3-multipart`, otherwise [`@uppy/aws-s3`][].
+We recomend setting [`shouldUseMultipart`][] option to use the right upload
+method depending on the file size.
```

Comment:
I'm not sure about the version number, if we remove them it makes for a much simpler sentence which still convays the message we want to pass, wdyt? All of our docs apply to up-to-date info, I feel like we mostly do not give any version info anywhere, it'd be a bit weird to start from here.

Reply:
Let's take the deprecation of `@uppy/aws-s3-multipart` to its own PR as I feel like there's going to be more work to get to a version we like. I've removed the mention of `@uppy/aws-s3` so this can land.